src/Microsoft.PowerShell.Commands.Management/commands/management/SetClipboardCommand.cs

```diff
@@ -44,6 +44,13 @@ public class SetClipboardCommand : PSCmdlet
         [Parameter]
         public SwitchParameter PassThru { get; set; }
 
+        /// <summary>
+        /// Gets or sets whether to use OSC52 escape sequence to set the clipboard of host instead of target.
+        /// </summary>
+        [Parameter]
+        [Alias("Remote")]
```

Comment:
@SteveL-MSFT Did the Cmdlet WG review and approve this alias?

Reply:
@dkaszews WG discussed this and believe there's too much ambiguity with the term `Remote` so we suggest the alias to be `ToLocalhost`

Reply:
@SteveL-MSFT Done